packages/earlybird/earlybird.0.1/opam

```diff
@@ -0,0 +1,34 @@
+opam-version: "2.0"
+synopsis: "OCaml debug adapter"
+description: """
+OCaml debug adapter.
+"""
+maintainer: "文宇祥 <hackwaly@qq.com>"
+authors: "文宇祥 <hackwaly@qq.com>"
+license: "MIT"
+homepage: "https://github.com/hackwaly/ocamlearlybird"
+bug-reports: "https://github.com/hackwaly/ocamlearlybird/issues"
+dev-repo: "git://github.com:hackwaly/ocamlearlybird.git"
+depends: [
+  "ocaml" {>= "4.06" & < "4.07"}
+  "dune"
```

Comment:
this needs to be flagged as a `{build}` dependency

Reply:
Fixed. Please return it upstream

Reply:
Should I file an another pull request? l'm unfamilar with opam publish work flow.